ajax/libs/jspanel/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jspanel",
+  "version": "2.6.3",
+  "filename": "jquery.jspanel.min.js",
+  "description": "A jQuery Plugin to create highly configurable multifunctional floating panels",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Flyer53/jsPanel.git"
+  },
+  "keywords": [
+    "jquery-plugin",
+    "jsPanel",
+    "panel",
+    "ui",
+    "layout",
+    "popup",
+    "widget",
+    "window",
+    "modal",
+    "tooltip",
+    "hint",
+    "bootstrap"
+  ],
+  "author": "Stefan Straesser <info@jspanel.de> (http://jspanel.de/)",
+  "homepage": "http://jspanel.de",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Flyer53/jsPanel.git",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
```

Comment:
Because we are going to add files from the zip files and the author said
> I plan to add the jsPanel3 versions to the "original" jsPanel repo in order to have every version in one repo. Also the jsPanel version 4 I'm already working on is planned to go to the "original" repo jsPanel.

I am thinking of that should we use the pattern for the current file structure in jsPanel3 repo?

Reply:
@PeterDaveHello @maruilian11 @pvnr0082t Could you give me some suggestion?
Thank you.

Reply:
I think we can follow the file structure in `jsPanel3` repo.

Reply:
@sufuf3 shouldn't you write down your thoughts first and then ask for suggestion?

Reply:
Sorry~
I'll follow the file structure in `jsPanel3` repo and update this PR.
Thanks

Reply:
@PeterDaveHello @maruilian11 @pvnr0082t 
May I ask that can we really let jsPanel follow jsPanel3 structure?
I know that we can't touch the existing files.
In this case, I use the soft links as the origin file structure.
But if I make jsPanel's file structure as the same with jsPanel3's, then many people can't use origin link to get the files, don't they?

Sorry for the bother, I think this case a little bit complicated.

Reply:
@PeterDaveHello @maruilian11 @pvnr0082t @extend1994 
May I ask that can we really let jsPanel follow jsPanel3 structure?
I know that we can't touch the existing files.
In this case, I use the soft links as the origin file structure.
But if I make jsPanel's file structure as the same with jsPanel3's, then many people can't use origin link to get the files, don't they?

Sorry for the bother, I think this case a little bit complicated.

Reply:
Directly discuss with me with well explained details next meeting please.

Reply:
OK thanks